doc/obs_integration.md

```diff
@@ -72,6 +75,108 @@ use `*.gem` files which are not supported by OBS services. It uses the
 [osc:commit](https://github.com/openSUSE/packaging_rake_tasks#osccommit) Rake
 task, same as the YaST packages.
 
+### Synchronizing GitHub Fork with OBS Branch
```

Comment:
I would move this section one level up to make it more visible. It is related to the OBS synchronization, sure, but it is more about "Building your own fork" :-)

But it is just a suggestion. If you decide to do it, please remove the "Details" header in the section below (keeping the content). Having a section with just a subsection looks weird.